projects/driver_controls/inc/power_distribution_controller.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+// Module for updating the power distribution state.
+//
+// Requires CAN and event_queue to be initialized.
+//
+// Chaos will ACK with an affirmative response if the state is valid.
```

Comment:
I don't think this is necessarily accurate anymore - it's just that you enqueue the sequence transition which may or may not succeed. In real use, any transition that driver controls requests should work since they should be synchronized. I'm just thinking of a case where driver controls somehow resets so the states desync. We should probably start driver controls with an "idle" message.